jobserver/views/job_requests.py

```diff
@@ -30,6 +30,10 @@ def filter_by_status(job_requests, status):
         "pending": lambda r: r.status.lower() == "pending",
         "succeeded": lambda r: r.status.lower() == "succeeded",
     }
+
+    if status not in status_lut:
+        return job_requests
+
     func = status_lut[status]
     return list(filter(func, job_requests))
```

Comment:
Is this equivalent to:

```
def filter_by_status(job_requests, status):
    if status not in ["failed", "running", "pending", "succeeded"]:
        # comment about why we might see an unexpected status
        return job_requests

    return [r for r in job_requests if r.status.lower() == status]
```

Reply:
Yep, that's exactly it, and that's much nicer, thanks!

Reply:
:cartwheeling: